cmd/api-router.go

```diff
@@ -311,7 +311,9 @@ func registerAPIRouter(router *mux.Router) {
 		// GetBucketReplicationConfig
 		router.Methods(http.MethodGet).HandlerFunc(
 			collectAPIStats("getbucketreplicationconfiguration", maxClients(httpTraceAll(api.GetBucketReplicationConfigHandler)))).Queries("replication", "")
-
+		// GetBucketReplicationStatus
+		router.Methods(http.MethodGet).HandlerFunc(
+			collectAPIStats("getbucketreplicationstatus", maxClients(httpTraceAll(api.GetBucketReplicationMetricsHandler)))).Queries("replication-metrics", "")
```

Comment:
```suggestion
			collectAPIStats("getbucketreplicationmetrics", maxClients(httpTraceAll(api.GetBucketReplicationMetricsHandler)))).Queries("replication-metrics", "")
```

Reply:
Can you move this all the way down to indicate these as extended API not part of AWS S3?